core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1646,73 +1649,46 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
+  private def makeGroupCoordinatorRequestContextFrom(
```

Comment:
the naming seems slightly off. how's createGroupCoordinatorRequestContext?

Reply:
the `From` part i guess. i don't think i've seen a method name like it, though it does make sense

Reply:
I can remove the `From` part. I am not too opinionated on this.